src/app/accessRequests/reviewOrganisationRequest.js

```diff
@@ -25,6 +25,7 @@ const get = async (req, res) => {
     request,
     selectedResponse: null,
     validationMessages: {},
+    currentPage: 'users',
```

Comment:
Need to pass this in the model for the post as well

Reply:
done